frontend/admin/src/js/components/PoolDashboard.tsx

```diff
@@ -37,291 +35,224 @@ import { UpdateSkillFamily } from "./skillFamily/UpdateSkillFamily";
 import SkillPage from "./skill/SkillPage";
 import { CreateSkill } from "./skill/CreateSkill";
 import { UpdateSkill } from "./skill/UpdateSkill";
-import HomePage from "./home/HomePage";
 import { Role } from "../api/generated";
 import DashboardPage from "./dashboard/DashboardPage";
 import ViewUser from "./user/ViewUser";
 
-const AdminNotAuthorized: React.FC = () => {
-  const intl = useIntl();
-  return (
-    <NotAuthorized
-      headingMessage={intl.formatMessage({
-        description:
-          "Heading for the message saying the page to view is not authorized.",
-        defaultMessage: "Sorry, you are not authorized to view this page.",
-      })}
-    >
-      <p>
-        {intl.formatMessage({
-          description:
-            "Detailed message saying the page to view is not authorized.",
-          defaultMessage:
-            "Oops, it looks like you've landed on a page that you are not authorized to view.",
-        })}
-      </p>
-    </NotAuthorized>
-  );
-};
-
-const routes = (
-  paths: AdminRoutes,
-  loggedIn?: boolean,
-  isAdmin?: boolean,
-): Routes<RouterResult> => [
+const routes = (paths: AdminRoutes): Routes<RouterResult> => [
   {
     path: paths.home(),
     action: () => ({
-      component: <HomePage />,
-      redirect: loggedIn ? paths.dashboard() : undefined,
+      redirect: paths.dashboard(),
     }),
   },
   {
     path: paths.dashboard(),
     action: () => ({
-      component: loggedIn ? <DashboardPage /> : <AdminNotAuthorized />,
+      component: <DashboardPage />,
+      authorizedRoles: [Role.Admin],
```

Comment:
How do people feel about defining the roles authorized to use the component right here?

Reply:
I personally like this. Not sure if there is even a better alternative outside of defining it in the component itself. 🤷

Reply:
i like it too. from doing a few react-router tutorials in the past, this seems to be the common place to specify :)

Reply:
Makes sense to me!